pkgs/tools/text/gripgen/default.nix

```
@@ -0,0 +1,25 @@
+{ boost, catch2, cmake, fetchFromGitHub, stdenv }:
+
+stdenv.mkDerivation rec {
+  pname = "gripgen";
```

```suggestion
  pname = "grip";
```

gripgen is only the tool to generate the database. (Also change commit and all-packages.nix)

---

Grip is already taken,
Its some gui app, that's why I used gripgen